mmv1/third_party/terraform/tests/resource_org_policy_policy.go

```diff
@@ -0,0 +1,535 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+// ----------------------------------------------------------------------------
+//
+//     ***     AUTO GENERATED CODE    ***    Type: DCL     ***
+//
+// ----------------------------------------------------------------------------
+//
+//     This file is managed by Magic Modules (https://github.com/GoogleCloudPlatform/magic-modules)
+//     and is based on the DCL (https://github.com/GoogleCloudPlatform/declarative-resource-client-library).
+//     Changes will need to be made to the DCL or Magic Modules instead of here.
+//
+//     We are not currently able to accept contributions to this file. If changes
+//     are required, please file an issue at https://github.com/hashicorp/terraform-provider-google/issues/new/choose
+//
+// ----------------------------------------------------------------------------
+
+package orgpolicy
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	dcl "github.com/GoogleCloudPlatform/declarative-resource-client-library/dcl"
+	orgpolicy "github.com/GoogleCloudPlatform/declarative-resource-client-library/services/google/orgpolicy"
+
+	"github.com/hashicorp/terraform-provider-google/google/tpgdclresource"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+)
+
+func ResourceOrgPolicyPolicy() *schema.Resource {
```

Comment:
Is there any particular reason you added the fields in a handwritten manner (meaning not adding them in OrgPolicy.yaml)? If you add them in OrgPolicy.yaml, this file should be generated automatically.

Reply:
Hi @hao-nan-li, The reason behind adding this file is that repository does not contains "resource_org_policy_policy.go" file nor the OrgPolicy.yaml  .

Our actual solution/modification is in the "resource_org_policy_policy.go" file that's why we added the file here.

Reply:
resource_org_policy_policy.go should be generated to Terraform-google-provider, you should not add  `resource_org_policy_policy.go`. 
https://googlecloudplatform.github.io/magic-modules/get-started/generate-providers/